core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -483,6 +483,12 @@ class ControllerApis(
         } else {
           new ApiError(TOPIC_AUTHORIZATION_FAILED)
         }
+      case ConfigResource.Type.CLIENT_METRICS =>
+        if (authHelper.authorize(requestContext, ALTER_CONFIGS, CLIENT_METRICS, resource.name)) {
```

Comment:
The KIP says that the ACL check is operation `ALTER` and resource `CLUSTER`. I think that perhaps the KIP should actually say `ALTER_CONFIGS` but the resource type should remain `CLUSTER`. OK?

Reply:
Corrected it.